src/Exception/RequestException.php

```diff
@@ -136,13 +139,18 @@ public static function getResponseBodySummary(ResponseInterface $response)
             return null;
         }
 
-        $summary = $body->read(120);
-        $body->rewind();
+        if (self::$responseSize) {
```

Comment:
This check is not needed if it is an integer. 

(Positive numbers will truncate, negative number will allow everything)?

Reply:
I changed the check for `null` only so that we get the full contents if the developer chooses to `setResponseSize(null);`